### PR TITLE
Ram 1500

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

**Comment by sshane** [comment]
Rough lat accel data from our drive:

Screenshot from 2022-06-16 00-14-51

File: selfdrive/car/chrysler/carstate.py [hunk 1]
```diff
@@ -3,150 +3,242 @@
...
-        ("BLIND_SPOT_LEFT", "BLIND_SPOT_WARNINGS"),
+        ("Blind_Spot_Monitor_Left", "BSM_1"),
+        ("Blind_Spot_Monitor_Right", "BSM_1"),
+      ]
+      checks += [("BSM_1", 2)]
+
+    if CP.carFingerprint in (CAR.RAM_1500, ):
+      signals += [
+        ("LKAS_Button", "Center_Stack_2"),  # LKAS Button
+        ("DASM_FAULT", "EPS_3"),  #EPS Fault
```
Comment (sshane) [inline selfdrive/car/chrysler/carstate.py:196]: @realfast is there a signal equivalent to this on other Chryslers?

File: selfdrive/car/chrysler/carstate.py [hunk 2]
```diff
@@ -3,150 +3,242 @@
...
-        ("BLIND_SPOT_LEFT", "BLIND_SPOT_WARNINGS"),
+        ("Blind_Spot_Monitor_Left", "BSM_1"),
+        ("Blind_Spot_Monitor_Right", "BSM_1"),
+      ]
+      checks += [("BSM_1", 2)]
+
+    if CP.carFingerprint in (CAR.RAM_1500, ):
+      signals += [
+        ("LKAS_Button", "Center_Stack_2"),  # LKAS Button
+        ("DASM_FAULT", "EPS_3"),  # EPS Fault
```
Comment (sshane) [inline selfdrive/car/chrysler/carstate.py:196]: @realfast is there a signal equivalent to this on other Chryslers?

Comment (realfast) [reply selfdrive/car/chrysler/carstate.py:196]: @sshane i assume you are referring to the DASM_Fault error? If so, yes that same fault exists. Check out the Ram_2500 DBC.

File: selfdrive/car/chrysler/carstate.py [hunk 3]
```diff
@@ -3,150 +3,211 @@
...
-from selfdrive.car.chrysler.values import DBC, STEER_THRESHOLD
+from selfdrive.car.chrysler.values import DBC, STEER_THRESHOLD, RAM_CARS
 
 
 class CarState(CarStateBase):
   def __init__(self, CP):
     super().__init__(CP)
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
-    self.shifter_values = can_define.dv["GEAR"]["PRNDL"]
+    self.shifter_values = can_define.dv["Transmission_Status"]["Gear_State"]
```
Comment (sshane) [inline selfdrive/car/chrysler/carstate.py:13]: We'll want to find a message that includes low, or maybe it's a bit in this message.

**Pull request merged**